src/SA/resources/entry.c

```diff
@@ -0,0 +1,48 @@
+#include <windows.h>
+#include "bofdefs.h"
+#include "base.c"
+
+#define DIV 1048576
+
+void getResources() {
+
+	// Get dat memory
+	MEMORYSTATUSEX statex;
+	statex.dwLength = sizeof(statex);
+
+	if (KERNEL32$GlobalMemoryStatusEx(&statex) == 0) {
+		BeaconPrintf(CALLBACK_ERROR, "Error fetching memory");
+		return;
+	}
+
+	internal_printf("Memory Used:\t%I64dMB/%I64dMB\n", (statex.ullTotalPhys - statex.ullAvailPhys) / DIV,
+		statex.ullTotalPhys / DIV);
+
+
+	// And now the primary disk
+	ULARGE_INTEGER totalBytes;
+	ULARGE_INTEGER freeBytes;
+
+	int a = KERNEL32$GetDiskFreeSpaceExA(NULL, NULL, &totalBytes, &freeBytes);
+	if (a == 0) {
+		BeaconPrintf(CALLBACK_ERROR, "Error fetching disk space");
+		return;
+	} else {
+		internal_printf("Free Space:\t%lu MB\n", freeBytes.QuadPart / DIV);
```

Comment:
Given what occurs with uptime may wonder why this works. Your divide happens to be by a power of 2 so it ends up being a shift instead of linking to that divide helper function, so this is fine

Reply:
After fighting uptime for a while, that is weird...